loctool.js

```diff
@@ -80,10 +80,20 @@ function usage() {
         "  Write all output to the given target dir instead of in the source dir.\n" +
         "-v or --version\n" +
         "  Print the current loctool version and exit\n" +
-        "-x or --xliffs\n" +
-        "  Specify the dir where the xliffs files live. Default: \".\"\n" +
+        "--projectType\n" +
+        "  the type of project, which affects how source files are read and resource files are written. Default: web \n" +
+        "--sourceLocale\n" +
+        "   Default locale of source string. (Default is en-US) \n" +
+        "--plugins\n" +
+        "  plugins to use that handle various file types in your project. \n" +
+        "--resourceFileTypes\n" +
+        "  Specifies the file type of the resource to be created. \n" +
         "--localizeOnly\n" +
         "  Generate a localization resource only. Do not create any other files at all after running loctool. \n" +
+        "--xliffResRoot\n" +
+        "  Specify the dir where the generation output should go. (Default is resources/) \n" +
+        "--xliffResName\n" +
```

Comment:
--xliffResName type1=defaultname1,type2=defaultname2,etc.

Reply:
Since the part that creates the resource has moved to each plugin, I changed the parameter name more appropriately.
something like: 
`--resourceFileNames json=strings.json --resourceDir json=output`